src/libraries/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -443,53 +454,29 @@ public void DirectorySecurity_CreateDirectory_NullSecurity()
         [Fact]
         public void DirectorySecurity_CreateDirectory_InvalidPath()
         {
-            DirectorySecurity security = new DirectorySecurity();
+            var security = new DirectorySecurity();
 
             Assert.Throws<ArgumentNullException>("path", () => security.CreateDirectory(null));
             Assert.Throws<ArgumentException>(() => security.CreateDirectory(""));
         }
 
-        [Fact]
-        public void DirectorySecurity_CreateDirectory_DefaultDirectorySecurity()
-        {
-            DirectorySecurity security = new DirectorySecurity();
-            Verify_DirectorySecurity_CreateDirectory(security);
-        }
-
-        [Theory]
-        [InlineData(FileSystemRights.ReadAndExecute, AccessControlType.Allow)]
-        [InlineData(FileSystemRights.ReadAndExecute, AccessControlType.Deny)]
-        [InlineData(FileSystemRights.WriteData,      AccessControlType.Allow)]
-        [InlineData(FileSystemRights.WriteData,      AccessControlType.Deny)]
-        [InlineData(FileSystemRights.FullControl,    AccessControlType.Allow)]
-        [InlineData(FileSystemRights.FullControl,    AccessControlType.Deny)]
-        public void DirectorySecurity_CreateDirectory_DirectorySecurityWithSpecificAccessRule(
-            FileSystemRights rights,
-            AccessControlType controlType)
-        {
-            DirectorySecurity security = GetDirectorySecurity(rights, controlType);
-            Verify_DirectorySecurity_CreateDirectory(security);
-        }
```

Comment:
Seems like these two tests are still missing.

Reply:
I improved this test and renamed it to DirectoryInfo_Create_MultipleAddAccessRules.

Reply:
What about DirectorySecurity_CreateDirectory_DefaultDirectorySecurity?

Reply:
Seems like DirectorySecurity_CreateDirectory_DefaultDirectorySecurity is testing the same as DirectoryInfo_Create_DefaultDirectorySecurity. How odd.